packages/stack/src/views/Header/HeaderBackground.tsx

```diff
@@ -1,8 +1,8 @@
 import * as React from 'react';
-import { View, StyleSheet, Platform, ViewProps } from 'react-native';
+import { Animated, StyleSheet, Platform, ViewProps } from 'react-native';
 
 export default function HeaderBackground({ style, ...rest }: ViewProps) {
-  return <View style={[styles.container, style]} {...rest} />;
+  return <Animated.View style={[styles.container, style]} {...rest} />;
```

Comment:
We're using Reanimated in stack. So we should change this to import `Animated` from `react-native-reanimated` instead.